docs/Documentation/run_RBM.md

```diff
@@ -106,3 +104,187 @@ All necessary files have been created at this point (see Appendix A for a descri
 
 where first **Salmon_0.5** refers to **Salmon_0.5_Network** ("**\_Network**" is appended by the model software) and second"**Salmon_0.5**" is the output file name (This refers to ***full_path/output_name*** . In this case, if no directory is addressed, the output file will be directly located in the current directory). "**Salmon_0.5.Spat**" is also created and is a file that cross-references element numbers in the output file "**Salmon_0.5.Temp**" to lat/long for
 post-processing.
+
```

Comment:
1) Fixed
2) Fixed
3) Good suggestion! I think this can be actually put at step 1!
4) There are details information about .Spat file and .Temp file at Appendix A.